### PR TITLE
Ft czar draw cards 150397541

### PR DESCRIPTION
**Pull request by QUDUSKUNLE**
####What does this PR do?
Czar should be able to draw cards by clicking

####What are the relevant pivotal tracker stories?
GIVEN user has been assigned the CZAR
WHEN user is to pick a card,
THEN they are prompted to click to draw the card as opposed to just randomly selecting the card and leaving the CZAR confused about what just happened.

####Screenshots (if appropriate)
screen shot 2017-09-11 at 2 18 25 pm

### CODE REVIEW

File: public/js/services/game.js [hunk 1]
```diff
@@ -27,6 +28,23 @@ angular.module('mean.system')
     var self = this;
     var joinOverrideTimeout = 0;
 
+    socket.on('player_limit_exceeded', function (data) {
+     const myModal = $('#playerRequirement');
```
[the remark this answers is not in the document]
Comment (mradeybee) [reply public/js/services/game.js:32]: @QUDUSKUNLE check your indentation here.

File: app/controllers/utils/sendEmail.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const nodemailer = require('nodemailer');
+/**
```
Comment (mradeybee) [inline app/controllers/utils/sendEmail.js:2]: Is another work merged into this PR?

**Review by mradeybee — CHANGES_REQUESTED** [review]
Fix the fixable hound ci complaints and also watch the indentations in your files.

**Comment by QUDUSKUNLE** [comment]
Noted!!!

**Pull request closed without merging**